xtra-libp2p/src/dialer.rs

```diff
@@ -46,6 +46,9 @@ impl Actor {
             .await
             .map_err(|_| Error::NoEndpoint)?;
 
+        dbg!(&listen_addresses);
+        dbg!(&self.connect_address);
```

Comment:
Are these meant to still be here? Do we not have a lint that detects this?

Reply:
we have a lint and they're not here as a result of the changeset. clippy doesn't run for each commit separately, if we care enough we could change our github actions, but I don't think it's worth the hassle.